pkg/storage/edv/encryptedformatter.go

```diff
@@ -58,9 +58,10 @@ func NewEncryptedFormatter(jweEncrypter jose.Encrypter, jweDecrypter jose.Decryp
 }
 
 // Format encrypts v into encrypted document format.
-func (f *EncryptedFormatter) Format(v []byte) ([]byte, error) {
+func (f *EncryptedFormatter) Format(k string, v []byte) ([]byte, error) {
```

Comment:
does it make sense to rename this back to `FormatPair`?

Reply:
Yes, I think it does. Just renamed it.